vmm/sandbox/src/stratovirt/devices/virtio_net.rs

```diff
@@ -62,9 +64,13 @@ pub struct VirtioNetDevice {
     pub(crate) fds: Vec<i32>,
     #[property(param = "netdev", predicate = "self.fds.len()<=0")]
     pub(crate) ifname: Option<String>,
-    #[property(param = "device", key = "bus", predicate = "self.addr.len()>0")]
+    #[property(
+        param = "device",
+        key = "bus",
+        predicate = "self.get_device_addr().len()>0"
+    )]
     pub(crate) pci_bus: Option<String>,
-    #[property(param = "device", predicate = "self.addr.len()>0")]
+    #[property(param = "device", predicate = "self.get_device_addr().len()>0")]
```

Comment:
if it is only for making clippy happy, maybe we can just remove the method `get_device_addr`。